webapp/portlet/src/main/webapp/vue-apps/profile-settings/components/ProfileSettings.vue

```diff
@@ -105,6 +105,7 @@ export default {
           this.getSettings();
         }
         this.displayMessage({type: 'success', message: this.$t('profileSettings.update.success.message')});
+        document.dispatchEvent(new CustomEvent('userPropertiesModified'));
```

Comment:
I am not sure how this event from the profile Settings will be triggered to the user profile which is in another page. Could you recheck ?

Reply:
done !